.github/workflows/pull-request-check.yml

```diff
@@ -0,0 +1,45 @@
+name: Pull Request Check
+on:
+  pull_request:
+    branches:
+      - trunk
+      - branch-2.7
+  check_run:
+    types: [created, rerequested, requested_action]
+
+jobs:
+  testing:
+    strategy:
+      matrix:
+        runs-on: [ubuntu-latest]
+        python-version: [2.7]
```

Comment:
should not it be 3.x ?

Reply:
Does Ambari support Python  version 3? :)